src/validationUtils.ts

```diff
@@ -5,3 +5,37 @@ export function validatePostalCode(postalCode: string): boolean {
 
   return validPostalCodes.includes(postalCode);
 }
+
+export function validateAMKA(amka: string | number): boolean {
+  // AMKA should be 11 digits long
+  const strAmka = amka.toString();
+  const regex = RegExp("^\\d{11}$");
+  if (regex.exec(strAmka) === null || strAmka == "00000000000") {
+    return false;
+  }
+  // The first 6 digits is the date-of-birth in DDMMYY format
+  if (parseInt(strAmka.substring(0, 2)) > 31) {
+    return false;
+  }
+  if (parseInt(strAmka.substring(2, 4)) > 12) {
+    return false;
+  }
+
+  // The last digit is a check digit cimputed using the Luhn algorithm
```

Comment:
@karate I believe you mean computed here instead of "cimputed"

Reply:
Indeed. Thanks.